### ISSUE TITLE
Metroid - Zero Mission (Europe): Sleep Mode non-functional

### ISSUE THREAD

**Issue by menguele** [mgba-emu/mgba#1482]
Upon using mGBA (0.7.2) this game's sleep mode doesn't work. It is unknow if it occur in this particular dump (EU).
Using 3DS' VC it seems to work normally.
I recorded a video and zipped it:
[MZM-1.zip](https://github.com/mgba-emu/mgba/files/3343653/MZM-1.zip)

**Comment by endrift** [comment]
What do you expect to happen in an emulator?

**Comment by menguele** [comment]
To Turn off display and sound output.

**Comment by endrift** [comment]
I'm not sure what that would accomplish in an emulator but I'll look into implementing it in the future.

**Comment by GValiente** [comment]
I was going to post a similar issue with sleep mode. On both VBA and NO$GBA both display and sound are turned off until wake up keys are pressed, but mGBA awakens without pressing anything.

**Comment by RetroEdit** [comment]
Yeah, I also noticed this in another game (Shrek 2), and thought it seemed odd. I kinda assumed games just did sleep mode by checking for the proper inputs each frame, but otherwise conserving battery by not doing anything the majority of each frame. Is sleep mode actually doing something more interesting to the hardware?
